src/EditorFeatures/Core/Implementation/Suggestions/SuggestedActionsSourceProvider.cs

```diff
@@ -220,6 +220,7 @@ private SuggestedActionSet InlineActions(SuggestedActionSet actionSet)
                         }
 
                         newActions.AddRange(childActionSets[0].Actions);
+                        continue;
                     }
 
                     newActions.Add(action);
```

Comment:
Stylistically, i prefer the above.  I literally cannot explain why as it's purely subjective.

If it's important to you, i can change though as i don't really care that much :)